<?xml version="1.0"?>
<launch>

    <group ns="uav0">
        <node pkg="tracking_uav_control" name="target_estimate_v1s1" type="target_estimate_v1s1" output="screen" />

        <!-- Target Estimation Initial Process Covariance -->
        <!-- x1, x2, x3, rq, vq -->
        <param name="P_init_0" value="0.8" />
        <param name="P_init_1" value="0.8" />
        <!-- param name="P_init_2" value="0.08" /-->
        <param name="P_init_3" value="90.0" />
        <param name="P_init_4" value="90.0" />
        <param name="P_init_5" value="90.0" />
        <param name="P_init_6" value="0.5" />
        <param name="P_init_7" value="0.5" />
        <param name="P_init_8" value="0.5" />

        <!-- Target Estimation Measurement Noise Covariance -->
        <!-- u, v, d, rc -->
        <param name="measurement_noise_0" value="5" />  
        <param name="measurement_noise_1" value="5" />
        <!-- param name="measurement_noise_2" value="0.0001" /-->
        <param name="measurement_noise_3" value="0.0001" />
        <param name="measurement_noise_4" value="0.0001" />
        <param name="measurement_noise_5" value="0.0001" />

        <!-- Target Estimation Initial Process Noise Covariance -->
        <!-- x1, x2, x3, rq, vq -->
        <param name="process_noise_0" value="0.0008" />
        <param name="process_noise_1" value="0.0008" />
        <!-- param name="process_noise_2" value="0.0008" /-->
        <param name="process_noise_3" value="0.05" />
        <param name="process_noise_4" value="0.05" />
        <param name="process_noise_5" value="0.05" />
        <param name="process_noise_6" value="0.05" />
        <param name="process_noise_7" value="0.05" />
        <param name="process_noise_8" value="0.05" />
    </group>

	

</launch>

